qa/test_template.sh

```diff
@@ -0,0 +1,28 @@
+topdir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )/..
+
+# Install dependencies
+# Note: glib-2.0 depends on python2, so reinstall the desired python afterward to make sure defaults are right
```

Comment:
Do we actually need this library? This hacky handling of Python versions has potential to screw us at some point.

Reply:
You can blame me for that particular bit of nonsense.  (It was already there long before this PR.)

Reply:
Do you remember why this was introduced?

Reply:
I would leave it now as it is. We can easily change it latter (as now it is only in one place).

Reply:
@ptrendx - I don't know why the dependency is there.  It was there long before I came to it.  I  agree with Janusz though; we can address it separately.